### ISSUE TITLE
Home: Image links do not have sufficient contrast

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#8]
**URL**: https://hcdev.openplus.ca/en
**User Agent:** Chrome/Firefox
**Language**: both
**Issue description**: The images for each link don't have enough contrast with the background and are difficult to see for some users. The contrast should be increased.
**Steps to reproduce**: Visual
![image](https://user-images.githubusercontent.com/65728700/126991258-ed2aaecb-99e0-4123-ae4c-721216abfaf8.png)

**Expected result**: The contrast should be minimum 3:1 with the background so that it's easier to see.

**Comment by ndolter** [comment]
Validated in both French and English that the images now pass the contrast contrast.
